.github/workflows/lint_actions.yml

```diff
@@ -37,4 +38,4 @@ jobs:
       - name: "Run actionlint"
         run: |
           echo "::add-matcher::.github/workflows/matchers/actionlint.json"
-          tools/actionlint.sh -color
+          tools/lint_actions.sh -color
```

Comment:
Let's use https://github.com/marketplace/actions/actionlint instead for CI

Reply:
Done. Tested. Matcher works seemlessly.